Bugsnag/Payload/BugsnagError.m

```diff
@@ -127,17 +127,25 @@ + (BugsnagError *)errorFromJson:(NSDictionary *)json {
 }
 
 - (void)updateWithCrashInfoMessage:(NSString *)crashInfoMessage {
-    @try {
-        // Messages that match this pattern should override the errorClass (and errorMessage if there is enough information.)
-        NSString *pattern = @"^(Assertion failed|Fatal error|Precondition failed): ((.+): )?file .+, line \\d+\n$";
-        NSRegularExpression *regex = [[NSRegularExpression alloc] initWithPattern:pattern options:NSRegularExpressionCaseInsensitive error:nil];
-        NSArray<NSTextCheckingResult *> *matches = [regex matchesInString:crashInfoMessage options:0 range:NSMakeRange(0, crashInfoMessage.length)];
+    NSArray<NSString *> *patterns = @[
+        // From Swift 2.2: https://github.com/apple/swift/blob/swift-2.2-RELEASE/stdlib/public/stubs/Assert.cpp#L24-L39
+        @"^(assertion failed|fatal error|precondition failed): ((.+): )?file .+, line \\d+\n$",
+        // From Swift 4.1: https://github.com/apple/swift/commit/d03a575279cf5c523779ef68f8d7903f09ba901e
+        @"^(Assertion failed|Fatal error|Precondition failed): ((.+): )?file .+, line \\d+\n$",
+        // From Swift 5.4: https://github.com/apple/swift/commit/1a051719e3b1b7c37a856684dd037d482fef8e59
```

Comment:
>  This update presents this information in a format matching the diagnostic conventions used by the compiler, which can be a slight productivity boost.

hahaha, well it does something nicer in vim at least, but that's an interesting reason to change a programming language 🙃